src/seedsigner/gui/screens/tools_screens.py

```diff
@@ -198,6 +198,92 @@ def update_title(self) -> bool:
 
 
 
+@dataclass
+class ToolsCoinEntropyEntryScreen(KeyboardScreen):
+    def __post_init__(self):
+        # Override values set by the parent class
+        # TRANSLATOR_NOTE: Inserts the number of flips required
+        self.title = _("Coin Flip 1/{}").format(self.return_after_n_chars)
+
+        # Specify the keys in the keyboard
+        self.rows = 2
+        self.cols = 2
+        self.key_height = GUIConstants.get_top_nav_title_font_size() + GUIConstants.EDGE_PADDING//4 + 2*GUIConstants.EDGE_PADDING
+        self.keys_charset = "".join([
+            "H",
+            "T",
+        ])
+
+        # Map Key display chars to actual output values
+        self.keys_to_values = {
+            "H": "1",
+            "T": "0",
+        }
+
+        # Now initialize the parent class
+        super().__post_init__()
+
+        self.components.append(TextArea(
+            text=_("(H)eads: 1   /   (T)ails: 0"),
+            screen_y = self.keyboard.rect[3] + 3*GUIConstants.COMPONENT_PADDING,
+        ))
+
+    def update_title(self) -> bool:
+        # TRANSLATOR_NOTE: Updates the title to show the current coin flip number and total flips required.
+        # {current} is the current flip number (1-based; e.g., 1, 2, 3, etc.).
+        # {total} is the total number of flips required
+        self.title = _("Coin Flip {current_flip_num}/{num_total_flips}").format(
+            current_flip_num=self.cursor_position + 1,
+            num_total_flips=self.return_after_n_chars
+        )
+        return True
+
+
+@dataclass
+class ToolsCoinEntropySetwiseEntryScreen(KeyboardScreen):
+    current_set: int = 1
+    total_sets: int = 12
+    required_bits: int = 11
+
+    def __post_init__(self):
+        current_set = self.current_set
+        total_sets = self.total_sets
+        
+        # TRANSLATOR_NOTE: 
+        # "current_set" = current set number (1-based)
+        # "total_sets" = total number of sets
+        self.title = _("Word {current_set}/{total_sets}").format(
+            current_set=current_set,
+            total_sets=total_sets
+        )
+        self.return_after_n_chars = self.required_bits
```

Comment:
I'm a bit torn here. On the one hand I think it's a little silly to pass in `required_bits` when it serves the exact same purpose as `return_after_n_chars`.

But on the other hand, the `required_bits` var name does help a bit to explain the context of what's going on and what's required in this particular coin flipping use case.

I think I slightly lean towards keeping `required_bits` (though that name could be improved... `num_flips_required`, `num_flips_needed`???) but basically kind of dislike either option so there's no ideal outcome here.